src/components/templates/NotFoundTemplate.js

```diff
@@ -9,7 +9,7 @@ import Heading from '../atoms/Heading';
 import MediumHeading from '../atoms/MediumHeading';
 import Anchor from '../atoms/Anchor';
 
-import CakeImg from '../../../static/assets/Group.png';
+import CakeImg from '../../../static/assets/Cake.png';
```

Comment:
remove this import, you take the picture from the json. also remov eit from the assets folder